screensharing/screenSharingTracker.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>Screen Sharing Tracker</title>
+  <style>
+    html {
+      background-color: #303135e6;
+    }
+
+    body {
+      align-items: center;
+      color: #e4e4e4;
+      display: flex;
+      justify-content: space-between;
+      margin: 0;
+      padding: 6px 16px;
+      -webkit-user-select: none;
+      -webkit-app-region: drag;
+    }
+
+    button {
+      background-color: #b8b8b8;
+      border: 0;
+      border-radius: 4px;
+      color: #282a2f;
+      height: 28px;
+      margin-right: 16px;
+    }
+
+    #text-container {
+      font-family:Arial, Helvetica, sans-serif;
+      font-size: 13px;
+      display: flex;
+    }
+
+    #double-line {
+      font-size: 15px;
+      font-weight:bolder;
+    }
+
+    #screen-share-marker-minimize {
+      color: #ababab;
+      font-family:Arial, Helvetica, sans-serif;
+      font-size: 13px;
+    }
+
+    #sharing-identity {
+      overflow: hidden;
+      text-overflow: ellipsis;
+      white-space: nowrap;
+      max-width: 170px;
+    }
+  </style>
+</head>
+
+<body>
+  <span id="double-line">‖</span>
+  <div id="text-container">
+    <span id="sharing-identity"></span><span id="static-text">&nbsp;is sharing your screen.</span>
+  </div>
+  <div>
+    <button id="screen-share-marker-stop">Stop sharing</button>
+    <a id="screen-share-marker-minimize">Hide</a>
```

Comment:
Could you please confirm that it is fine to have not translatable texts here?

Reply:
Honestly I haven't considered that, as it stands I didn't find any support for translating in the project i assumed it was just accepted as is.
I'm guessing changing these strings to parameters would do the trick. So that each application can just pass whatever they want, what do you think?

Reply:
Yes we don't have translation support here. Passing the strings is too much customisation for me :) Also it won't be so easy because you will have to add support for changing them realtime for the use case when the user changes the language. Let's take this offline and decide what to do!